changelogs/fragments/638-ec2_security_group_group_name_datatype.yml

```diff
@@ -0,0 +1,2 @@
+trivial:
+- set type as raw for rules->group_name as it can accept both str and list.
```

Comment:
```suggestion
minor_changes:
- ec2_security_group - set type as ``raw`` for rules->group_name as it can accept both ``str`` and ``list`` (https://github.com/ansible-collections/amazon.aws/pull/971).
```